valence_protocol/src/packets/s2c.rs

```diff
@@ -185,6 +185,13 @@ pub mod play {
         pub reset: bool,
     }
 
+    #[derive(Copy, Clone, Debug, Encode, Decode, Packet)]
+    #[packet_id = 0x10]
+    pub struct CloseContainerS2c {
+        /// Ignored by notchian clients.
+        pub window_id: u8,
+    }
+
```

Comment:
Looks like this packet has not been added to the `S2cPlayPacket` enum at the bottom of the module.